nucleus/grizzly/config/src/main/java/org/glassfish/grizzly/config/SSLConfigurator.java

```diff
@@ -252,23 +252,27 @@ protected SSLContext initializeSSLContext() {
                 
                 //key store settings
                 setAttribute(serverSF, "keystore", ssl.getKeyStore(), "javax.net.ssl.keyStore", null);
+                setAttribute(serverSF, "additionalKeystores", ssl.getKeyStore(), "fish.payara.ssl.additionalKeyStores", null);
```

Comment:
should be refactored into a static constant

Reply:
_Referring to all suggestions in SSLConfigurator:_ I have only added my two additional properties into these existing code blocks trying to keep with the style of code that was already there. In my opinion it would be more strange to just change these parts to static constants and leave the rest as strings. I could change all of these to use static constants but then I'm changing code that's nothing to do with my ticket, but if we think that's best I can go ahead and do that

Reply:
IMHO refactoring legacy code into better style is part of what we are supposed to be doing as we see it.